python/tvm/relay/op/_transform.py

```diff
@@ -1,8 +1,51 @@
 #pylint: disable=invalid-name, unused-argument
 """Backend compiler related feature registration"""
 from __future__ import absolute_import
+import topi
+import topi.cuda
+from tvm import container
 from . import op as _reg
-from .op import schedule_injective
+from .op import (schedule_injective, register_compute, register_schedule,
+                 register_pattern, OpPattern)
+
+schedule_broadcast = schedule_injective
+
+# squeeze
+@register_compute("squeeze")
+def squeeze_compiler(attrs, inputs, output_type, target):
+    """Compiler for squeeze dims."""
+    assert len(inputs) == 1
+
+    if attrs.axis is None:
+        axis = None
+    elif isinstance(attrs.axis, container.Array):
+        axis = tuple(attrs.axis)
+    else:
+        axis = int(attrs.axis)
```

Comment:
Register compute from [cpp code ](https://github.com/dmlc/tvm/blob/401ffe131e207bc83fa424df3dbc14ed1c987731/src/relay/op/tensor/transform.cc#L831) using `set_attr<FTVMCompute> ` is better?